tests/conftest.py

```diff
@@ -1,3 +1,4 @@
+# type: ignore
```

Comment:
Ignore what and why?
Please use the `# type: ignore[code]` format and include a comment explaining why (if it is not obvious).

Reply:
For the why part (from the initial PR description, section 'Things I've ignored'):

```
- tests: functions decorated with @pytest.fixture; for two reasons. The decorator is untyped, making the decorated functions also untyped. I took a few shots at writing a stub for the decorator with no success. Second, the actual fixture typing is not utilized anywhere, since its result is injected in the related tests and properly typed in the test function's definition.
- tests: conftest.py => ignored in it's entirety. Everything is a fixture.
```

But I think I saw some on some of your notes something for the fixtures so I will revisit my approach on that (and make sure I don't leave any other 'bare' type ignores)